report/5.Solution/FromScreenToMotors.tex

```diff
@@ -1,85 +1,77 @@
 \section{Mapping between computer vision and movement}
 Two functions are needed to translate the screen position into a velocity on the motors.
-First we need to transfer the screen position to the USB transfer, and then that format to a motor velocity.\todo{Should this be velocity or speed?}
+The first task is to translate the screen position into a format usable for the USB transfer, and then transform that format into a motor velocity.\todo{Should this be velocity or speed?}
 
 \subsection{Screen to motors}
-In order to translate the position on the screen to a velocity two functions are needed.
-First we need to transfer the screen position to the USB transfer, and then that format it to a motor velocity.
+To translate the position on the screen into a velocity two functions are needed.
+First of all, it is necessary to make a function maps the numbers correctly. \todo{Which numbers precisely?}
 
-We need to make a function on the python part that maps these numbers.
+This function will require a 2d vector named \textit{pos} as input. \todo{what is pos}
 
-This function will take in 'pos` which is going to be a 2d vector.
+The next goal is to take a location on the screen and translate it into a 8-bit value. \todo{why} 
+Due to the resolution of the camera that is used, the range on the screen is $0 - 480$ on one axis and $0 - 640$ on the other.
```

Comment:
Maybe specify what axis' they are. X and Y are pretty universal. Alternatively vertically and horizontally.